internal/serve/graphql/resolvers/utils.go

```diff
@@ -50,3 +167,39 @@ func getColumnMap(model any) map[string]string {
 	}
 	return fieldToColumnMap
 }
+
+func parsePaginationParams(first *int32, after *string, last *int32, before *string, defaultLimit int32) (PaginationParams, error) {
+	if first != nil && last != nil {
+		return PaginationParams{}, fmt.Errorf("first and last cannot be used together")
+	}
+
+	if after != nil && before != nil {
+		return PaginationParams{}, fmt.Errorf("after and before cannot be used together")
+	}
+
+	var cursor *string
+	var limit int32
+	var isForward bool
+	if first != nil {
+		cursor = after
+		limit = *first
+		isForward = true
+	} else if last != nil {
+		cursor = before
+		limit = *last
+		isForward = false
+	} else {
+		limit = defaultLimit
+	}
```

Comment:
If I omit `first` and `last`, is `isForward` `false` because its not explicitly defined?

Reply:
@JakeUrban Yes it would be false. However, now that I think about it, it should be true so by default its returning from decreasing -> increasing transaction IDs 🤔

Reply:
Also changed the default order to descending